io.github.cges30901.hmtimer.json

```diff
@@ -0,0 +1,74 @@
+{
+  "app-id": "io.github.cges30901.hmtimer",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "hmtimer",
+  "finish-args": [ 
+    "--socket=x11",
+    "--share=ipc",
+    "--device=dri",
+    "--filesystem=~/.config/autostart",
```

Comment:
This permission does just bypass the entire sandbox, so ideally you don't grant it and just tell users how to configure autostart on their desktop.

Reply:
OK, I'll remove it and think about ho to do it correctly.